src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$stopwatch-size: 80vmin;
+$center-size: $stopwatch-size / 20;
+$minute-hand-heigth: $stopwatch-size / 4;
```

Comment:
There is a typo in the variable name $minute-hand-heigth. It should be $minute-hand-height.